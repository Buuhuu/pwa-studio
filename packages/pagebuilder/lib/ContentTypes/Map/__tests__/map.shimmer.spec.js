import React from 'react';

import { createTestInstance } from '@magento/peregrine';

import MapShimmer from '../map.shimmer';

jest.mock('@magento/venia-ui/lib/classify');

test('renders an empty MapShimmer component', () => {
    const component = createTestInstance(<MapShimmer />);

    expect(component.toJSON()).toMatchSnapshot();
});

test('renders a MapShimmer component with height', () => {
    const props = {
        height: '400px'
    };
    const component = createTestInstance(<MapShimmer {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
});
