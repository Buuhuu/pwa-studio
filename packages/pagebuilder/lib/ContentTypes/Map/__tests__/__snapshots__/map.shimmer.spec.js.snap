// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a MapShimmer component with height 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "border": undefined,
      "borderWidth": undefined,
      "height": "400px",
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
/>
`;

exports[`renders an empty MapShimmer component 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "border": undefined,
      "borderWidth": undefined,
      "height": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
/>
`;
